README.md

```diff
@@ -78,29 +78,29 @@ pip install .
 With singularity installed run the following to built the xgboost container
 
 ```bash
-cd hpolib/container/recipes/ml
+cd hpobench/container/recipes/ml
 sudo singularity build xgboost_benchmark Singularity.XGBoostBenchmark
 ```
 
 You can use this local image with:
 
 ```python
-from hpolib.container.benchmarks.ml.xgboost_benchmark import XGBoostBenchmark
+from hpobench.container.benchmarks.ml.xgboost_benchmark import XGBoostBenchmark
 b = XGBoostBenchmark(task_id=167149, container_name="xgboost_benchmark", 
-                     container_source='./') # path to hpolib/container/recipes/ml
+                     container_source='./') # path to hpobench/container/recipes/ml
 config = b.get_configuration_space(seed=1).sample_configuration()
 result_dict = b.objective_function(config, fidelity={"n_estimators": 128, "subsample": 0.5})
 ```
 
 ### Remove all caches
 
-#### HPOlib data
-HPOlib stores downloaded containers and datasets at the following locations:
+#### HPOBench data
+HPOBench stores downloaded containers and datasets at the following locations:
 
 ```bash
-$XDG_CONFIG_HOME # ~/.config/hpolib2
-$XDG_CACHE_HOME # ~/.cache/hpolib2
```

Comment:
Sorry, I replaced this directly on master. There should be 3 different directories.